<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>293</width>
    <height>92</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout" columnstretch="0,1">
     <item row="0" column="1">
      <widget class="QSpinBox" name="xInput">
       <property name="minimum">
        <number>-2000000000</number>
       </property>
       <property name="maximum">
        <number>2000000000</number>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>X:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Z:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QSpinBox" name="zInput">
       <property name="minimum">
        <number>-2000000000</number>
       </property>
       <property name="maximum">
        <number>2000000000</number>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QSpinBox" name="yInput">
       <property name="minimum">
        <number>-2000000000</number>
       </property>
       <property name="maximum">
        <number>2000000000</number>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Y:</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
  <zorder>xInput</zorder>
  <zorder>xMaxInput</zorder>
  <zorder>label</zorder>
  <zorder>label_2</zorder>
  <zorder>yInput</zorder>
  <zorder>yMaxInput</zorder>
  <zorder>zMaxInput</zorder>
  <zorder>zInput</zorder>
  <zorder>label_3</zorder>
  <zorder>stackedWidget</zorder>
  <zorder></zorder>
  <zorder>zMaxInput</zorder>
  <zorder>yMaxInput</zorder>
  <zorder>xMaxInput</zorder>
  <zorder>widthLabel</zorder>
  <zorder>heightLabel</zorder>
  <zorder>lengthLabel</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
